bucket/emplace.json

```
@@ -0,0 +1,21 @@
+{
+  "homepage": "https://github.com/tversteeg/emplace",
+  "version": "0.2.11",
+  "description": "Command-line tool to mirror installed software on multiple machines.",
+  "license": "GPL-3.0",
```

https://github.com/tversteeg/emplace/blob/8cdea75c3d61d20f5021062c17124b11fd6c773e/Cargo.toml#L9

```suggestion
    "license": "AGPL-3.0-or-later",
```